packages/honeylemonjs/src/HoneylemonService.ts

```diff
@@ -815,7 +815,8 @@ class OrderbookService {
 
   async getOpenOrders(makerAddress) {
     const ordersResponse = await this.apiClient.getOrdersAsync({
-      makerAddress: makerAddress.toLowerCase()
+      makerAddress: makerAddress.toLowerCase(),
+      makerAssetData: this.makerAssetData
```

Comment:
This is in order to be able to store multiple contracts (e.g. BTC, ETC) in the same 0x API in the future